front/socket.form.php

```diff
@@ -51,7 +51,7 @@
 }
 
 $socket = new Socket();
-if (isset($_POST["add"])) {
+if (isset($_POST["add"]) && !isset($_POST["execute_multi"])) {
```

Comment:
Unless I am wrong, it should be possible to mutualize here code from
`if (isset($_POST["add"]) || isset($_POST["execute_single"]) || isset($_POST["execute_multi"])) {`

Reply:
done